var/spack/repos/builtin/packages/rnaz/package.py

```diff
@@ -11,5 +11,11 @@ class Rnaz(AutotoolsPackage):
 
     homepage = "https://www.tbi.univie.ac.at/software/RNAz"
     url      = "https://www.tbi.univie.ac.at/software/RNAz/RNAz-2.1.tar.gz"
+    git      = "https://github.com/ViennaRNA/RNAz.git"
 
+    version('2.1.1', commit='f2c19f7237f2eb3df04f4747c8c11616447ec095')
     version('2.1', sha256='b32ec0361889319f2058f224d6c456c853dbc30dff4dba90c53a8f9fd7b83be5')
+
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
```

Comment:
Shouldn't we:
```suggestion
    with when('@2.1.1:'):
        depends_on('autoconf', type='build')
        depends_on('automake', type='build')
        depends_on('libtool', type='build')
```
if the package ships with a `configure` in v2.1?